### PR TITLE
Added iBridge module

### PR DESCRIPTION
**Pull request by tuxudo**
As requested by @gmarnin, who also helped with debugging :) Resolves issue #1113

### CODE REVIEW

**Comment by gmarnin** [comment]
I tested this module on the iMac Pro and MacBook Pro 2018. The info on the client tab looks good. I wasn't able to test the dashboard widget or listings page. I suspect the error I get for those pages is due to an unrelated database issue I'm having.

**Comment by gmarnin** [comment]
I updated my db and I can now see the iBridge listing view. Looks good. 
Dashboard widget looks good too.

File: app/modules/ibridge/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+iBridge Module
+==============
+
+Details information about the iBridge in the Mac.
+
+Database:
+* ibridge_boot_uuid - VARCHAR(255) - Boot UUID of iBridge
```
Comment (bochoven) [inline app/modules/ibridge/README.md]: Please remove the ibridge prefix for the table names - too much duplication

File: app/modules/ibridge/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+* ibridge_build - VARCHAR(255) - iBridge OS build
+* ibridge_model_identifier - VARCHAR(255) - Model ID of iBridge
+* ibridge_model_name - VARCHAR(255) - Model name of iBridge
+* ibridge_serial_number - VARCHAR(255) - iBridge serial number
```
Comment (bochoven) [inline app/modules/ibridge/README.md]: Maybe just keep the ib_serial_number

File: app/modules/ibridge/README.md (hunk 1 above)
Comment (tuxudo) [reply app/modules/ibridge/README.md]: Prefixes removed

File: app/modules/ibridge/README.md (hunk 2 above)
Comment (tuxudo) [reply app/modules/ibridge/README.md]: Removed prefixes, except for ibridge_serial_number

**Pull request merged**